data/assets/07_matrix_view.css

```diff
@@ -1,3 +1,10 @@
+:root {
+  --matrix-border-color: lightgray;
+  --matrix-agent-bg-color: #eeeeee;
+  --matrix-diagonal-bg-color: white;
+  --matrix-data-width: 150px;
+}
+
 .a-data {
```

Comment:
We will need to make element classnames little bit more user friendly, so that customers can easily deduce to which element the style applies (ideally without having to look at the Python code). For example, I have no idea to which element this applies. It is IMHO easier to make the classnames descriptive than to provide documentation describing which is which.
It is also good to make them consistent, so that everything related to say matrix view, has `matrix` in the name. So for example `chart_legend__<whatever>` should be `matrix_legend__<whatever>`.

Reply:
Will work on naming, good point!